code/controllers/subsystems/initialization/computer_networks.dm

```diff
@@ -0,0 +1,37 @@
+SUBSYSTEM_DEF(networking)
+	name = "Computer Networks"
+	init_order = SS_INIT_COMPUTER_NETS
```

Comment:
Doesn't init, so give it the no init flag and remove this.

Reply:
don